marklogic-data-hub-central/ui/e2e/cypress/fixtures/users/developer.json

```diff
@@ -1,6 +1,17 @@
 {
     "user-name": "hc-developer",
-    "description": "A data hub dev user",
+    "description": "A hub central developer user",
     "password": "password",
-    "role": ["data-hub-developer", "hub-central-mapping-reader", "hub-central-saved-query-user"]
-}
\ No newline at end of file
+    "role": [
```

Comment:
@ryanjdew Is hub-central-developer coming soon??

Reply:
It is comming in this sprint